scss/_variables.scss

```diff
@@ -640,7 +640,7 @@ $input-bg:                              $white !default;
 $input-disabled-bg:                     $gray-200 !default;
 $input-disabled-border-color:           null !default;
 
-$input-color:                           $gray-700 !default;
+$input-color:                           $body-color !default;
```

Comment:
Just to confirm that this change is intended here?

Reply:
Yup, decided there's no good reason to lighten the color or detach it from the body.